src/datasource/components/QueryEditor.tsx

```diff
@@ -265,7 +281,7 @@ export class QueryEditor extends PureComponent<Props, State> {
   onIntervalChange = (value?: string) => {
     const { onChange, query, onRunQuery } = this.props;
     // not sending input less than 5 secs
-    if (Number(value) < 5) {
+    if (value?.length && +value < 5) {
```

Comment:
is there a benefit to this over `Number()`?

Reply:
Number() is pretty much equivalent to +, but it is shorter and I like it 🤷 

https://i.stack.imgur.com/LLrgj.png